mon-pix/app/routes/campaigns/assessment/skill-review.js

```diff
@@ -20,6 +20,11 @@ export default class SkillReviewRoute extends Route {
         userId: user.id,
       });
       const trainings = await campaignParticipation.hasMany('trainings').reload();
+
+      /* Je reload le model de l'utilisateur quand j'ai des "trainings" &
+       "hasRecommendedTrainings" est à false*/
```

Comment:
Je propose une vision plus fonctionnelle

```suggestion
      // Reload the user to display my trainings link on the navbar menu
```

(attention au lint, la ligne doit être trop longue)